mrchecker-framework-modules/mrchecker-webapi-module/src/main/java/com/capgemini/mrchecker/webapi/pages/httbin/SimpleXMLPage.java

```diff
@@ -0,0 +1,45 @@
+package com.capgemini.mrchecker.webapi.pages.httbin;
+
+import com.capgemini.mrchecker.webapi.core.BasePageWebAPI;
+import com.capgemini.mrchecker.webapi.core.base.driver.DriverManager;
+import com.capgemini.mrchecker.webapi.pages.environment.GetEnvironmentParam;
+
+import io.restassured.response.Response;
+import org.json.XML;
+
+import java.sql.Driver;
+
+public class SimpleXMLPage extends BasePageWebAPI {
+	
+	private final static String	HOSTNAME	= GetEnvironmentParam.HTTPBIN.getValue();
+	private final static String	PATH		= "/xml";
+	private final static String	ENDPOINT	= HOSTNAME + PATH;
+	//description of XML "Page"
+	private final static String XML_DOCUMENT_ROOT 	= "slideshow";
+	private final static String XML_ELEMENT_ITEM = "item";
+	private final static int ITEMS_NUMBER 	= 3;
+	private final static String XML_ELEMENT_DATE = "date";
+	private final static String XML_ELEMENT_DATE_VALUE = "Date of publication";
+	private final static String XML_ENCODING = "us-ascii";
+
+	public static String getXmlDocumentRoot(){	return XML_DOCUMENT_ROOT;	}
+
+	public static int getItemsNumber() { return ITEMS_NUMBER;	}
```

Comment:
I think those methods should return values read from XML document instead of hard-coded values

Reply:
I moved them to Test Class since initially it was meant to describe "Page" as I though it was a static object. But it is a valid point it should be place in Test Class and only general descriptors e.g. Attrribute and Element keys should be in Page. I created enums for them.